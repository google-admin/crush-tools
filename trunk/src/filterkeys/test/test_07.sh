test_number=07
description="combined columns make similar keys"

outfile="$test_dir/test_$test_number.actual"
expected="$test_dir/test_$test_number.expected"

$bin -p -K key1,key2 -f "$test_dir/test-filter-3.in" "$test_dir/test-3.in" > "$outfile"

if [ $? -ne 0 ] ||
   [ "`diff -q $outfile $expected`" ]; then
  test_status $test_number 1 "$description" FAIL
else
  test_status $test_number 1 "$description" PASS
  rm "$outfile"
fi
