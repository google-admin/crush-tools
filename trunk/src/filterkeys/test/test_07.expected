value1^key1^key2^value2
3491^ccv^aab^56.341
3491^ddd^erb^56.341
3491^wwz^awz^98.765
5671^ccv^aab^56.341
3491^ddd^erb^56.341
3451^wwz^awz^98.765
